content/experiments/animated-screen/styles.css

```
@@ -0,0 +1,211 @@
+
+  --blue-rgb: 33 150 243;
+  --primary-rgb: var(--blue-rgb);
+
+  width: 100%;
+  background-color: rgb(var(--background-rgb));
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  padding: 60px 0;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)